### PR TITLE
Set `xtra_libp2p` log level to `Info`

### PR DESCRIPTION
**Pull request by luckysori**

### CODE REVIEW

**Review by klochowicz — APPROVED** [review]

**Review by da-kami — APPROVED** [review]
I think `info` as default should not be too spammy.

**Comment by klochowicz** [comment]
bros r+

File: shared-bin/src/logger.rs [hunk 1]
```diff
@@ -33,7 +33,8 @@ pub fn init(level: LevelFilter, json_format: bool) -> Result<()> {
         .add_directive("_=off".parse()?) // rocket logs headers on INFO and uses `_` as the log target for it?
         .add_directive("rocket=off".parse()?) // disable rocket logs: we have our own
         .add_directive("xtra=warn".parse()?)
-        .add_directive("sled=warn".parse()?); // downgrade sled log level: it is spamming too much on DEBUG
+        .add_directive("sled=warn".parse()?) // downgrade sled log level: it is spamming too much on DEBUG
+        .add_directive("xtra_libp2p=info".parse()?);
```
Comment (bonomat) [inline shared-bin/src/logger.rs:37]: Note: if you set it here to info you can't set it to a different level using `RUST_LOG`!

**Pull request merged**